index.html

```diff
@@ -30,7 +30,7 @@
   <header>
     <a href="/tz-registrace">
       <div class="container alert alert-primary" role="alert">
-        Nové politické hnutí Budoucnost je registrováno a&nbsp;připravuje se na volební kampaň
+        Ustavující sjezd nového politického hnutí Budoucnost se odehraje v&nbsp;sobotu 27.6. v&nbsp;Praze.
```

Comment:
[Číslovky označující části data jsou běžně odděleny mezerou](https://prirucka.ujc.cas.cz/?id=810) – [nezalomitelnou mezerou.](https://prirucka.ujc.cas.cz/?id=880) Je možné je psát i dohromady, to je ale objevuje spíš v odborných a technických kontextech. 

```suggestion
        Ustavující sjezd nového politického hnutí Budoucnost se odehraje v&nbsp;sobotu 27.&nbsp;6. v&nbsp;Praze.
```